src/recensio/plone/browser/canonical.py

```diff
@@ -0,0 +1,24 @@
+from plone.registry.interfaces import IRegistry
+from zope.component import getUtility
+from zope.interface import Interface
+
+
+class IRecensioSettings(Interface):
+    """XXX import the proper interface once we have a control panel."""
+
+
+class CanonicalURLHelper:
```

Comment:
I‌ also started on this, but then I‌ implemented the RecensioSettings controlpanel first.
I thought of a method in `recensio.plone.utils` instead of a mixin class.

what do you think?

Reply:
I do not like both :)